principal_travel/templates/travel/trip_list.html

```diff
@@ -26,6 +26,35 @@ <h4 class="panel-title">
   </div>
 </div>
 
+<h1>Filters: </h1>
+
+<table id="filter_table">
+  <tr>
+    <td class="col-md-2"><p> Principal Title : </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="principal_list"> {{ principal_title }} </p></strong></td> 
+    <td class="col-md-2"><p> Year : </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="year">{{ year }}</p></strong></td>
+  </tr>
+  <tr>
+    <td class="col-md-2"><p> Principal Last Name: </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="principal_name"> {{ principal_name }} </p></strong></td>
+    <td class="col-md-2"><p> Region : </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="region">{{ region }}</p></strong></td>
+  </tr>
+  <tr>
+    <td class="col-md-2"><p> Quick Dates: </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="quick_dates">{{ quick_dates }}</p></strong></td> 
+    <td class="col-md-2"><p> Country : </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="country">{{ country }} </p></strong></td>
+  </tr>
+  <tr>
+    <td class="col-md-2"><p> Month : </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="month">{{ month }}</p></strong></td>
+    <td class="col-md-2"><p> Event Type : </p></td>
+    <td class="col-md-4 text-uppercase"><strong><p id="event_type">{{ event_type }} </p></strong></td>
+  </tr>
+</table>
+
```

Comment:
Very minor detail: can you remove the extra space between the word and the semicolon? That appears on lines 33, 35, 41, 47, 51, 53. So instead of month : it would be month: